dialog-el.html

```diff
@@ -6,11 +6,19 @@
     :host {
       position: absolute;
       display: inline-block;
+      transform: translate(0,0);
     }
     :host([hidden]) {
       display: none !important;
     }
-    
+    :host([attach]) {
```

Comment:
I like the idea of this `attach` property, and doing it in CSS makes it so the UI updates seamlessly as the value changes.

From a naming standpoint `attach` doesn't really represent this functionality well (and it isn't documented which we need). Can we rename this to `arrow-direction` or something of the like and get it added to the docs?